app/src/main/resources/html/main.js

```diff
@@ -50,7 +52,7 @@ async function main()
 
         // Render all elements!
         for (const elem of sceneContent.getElems()) {
-            elem.render(ctx, transform);
+            elem.render(ctx, transform)
```

Comment:
```suggestion
            elem.render(ctx, transform);
```

Let's leave the `;` here.